packages/openid4vc/tests/openid4vc.e2e.test.ts

```diff
@@ -97,7 +98,7 @@ describe('OpenId4Vc', () => {
                       method: 'did',
                       didUrl: verificationMethod.id,
                     },
-                    disclosureFrame: { university: true, degree: true },
+                    disclosureFrame: { _sd: ['university', 'degree'] } as DisclosureFrame<SdJwtVcPayload>,
```

Comment:
This is because the type of disclosureFrame in our package is set to a generic type created based on the type of claim.
I think we can do one of:

- Add generic in openid4vc type.
- Add `as` in somewhere middle (e.g.  sdJwtVcService.ts)